core/field_colour.ts

```diff
@@ -399,22 +394,22 @@ export class FieldColour extends Field<string> {
   private onMouseMove_(e: PointerEvent) {
```

Comment:
Remove underscore. Here and on all other `private` methods throughout this file..